static/app/views/settings/organizationMembers/inviteBanner.tsx

```diff
@@ -1,35 +1,90 @@
-import {useState} from 'react';
+import {useCallback, useEffect, useState} from 'react';
 import styled from '@emotion/styled';
 
+import {promptsCheck, promptsUpdate} from 'sentry/actionCreators/prompts';
+import Feature from 'sentry/components/acl/feature';
```

Comment:
Instead of wrapping in a feature, you can just check `organization.features` and just return null